### PR TITLE
feat: tidb dashboard for clinic

### PR DESCRIPTION
**Pull request by baurine**
What did:
- Make tidb dashboard reusable for clinic

### CODE REVIEW

File: ui/packages/tidb-dashboard-for-clinic/.env.production [hunk 1]
```diff
@@ -0,0 +1,5 @@
+REACT_APP_VERSION=$npm_package_version
+REACT_APP_MIXPANEL_HOST=https://telemetry.pingcap.com/api/v1/dashboard/report
+# TODO: Move to GitHub Secrets
+# DO NOT COPY THE TOKEN TO OTHER PROJECTS
+REACT_APP_MIXPANEL_TOKEN=
```
Comment (shhdgit) [inline ui/packages/tidb-dashboard-for-clinic/.env.production:5]: How do we distinguish tokens of different projects in a monolithic dashboard repository?

File: ui/packages/clinic-client/swagger/spec.json [hunk 2]
```diff
@@ -0,0 +1,180 @@
+{
```
Comment (shhdgit) [inline ui/packages/clinic-client/swagger/spec.json:1]: This means we need to sync the clinic or cloud api specs from those projects. If we still need to maintain the multi projects in separate repos, why don't we put the dashboard-for-x codes in the `x` repository?

File: ui/packages/tidb-dashboard-for-clinic/src/client/index.tsx [hunk 3]
```diff
@@ -0,0 +1,124 @@
+import React from 'react'
```
Comment (shhdgit) [inline ui/packages/tidb-dashboard-for-clinic/src/client/index.tsx:1]: Like swagger specs, we still need many duplicated codes between project `x` and dashboard-for-x.

File: ui/packages/tidb-dashboard-for-clinic/.env.production (hunk 1 above)
Comment (baurine) [reply ui/packages/tidb-dashboard-for-clinic/.env.production:5]: oh, I missed its value.

**Pull request merged**